website/versioned_docs/version-5.x/api.md

```diff
@@ -42,6 +44,39 @@ Note that if an application is registered from within another application, that
 
 `undefined`
 
+### Configuration object
+```js
+singleSpa.registerApplication({
+	name: 'appName',
+	app: () => System.import('appName'),
+	activeWhen: '/appName'
+	customProps: {}
+})
+```
+
+<h3>arguments</h3>
+
+<dl className="args-list">
+	<dt>name?: string</dt>
```

Comment:
The question mark here makes it seem like name is optional, but it isn't. Please remove:

```suggestion
	<dt>name: string</dt>
```

Same goes for the other properties that are required. The `customProps` are optional, but everything else is required.